demo/src/screens/__tests__/__snapshots__/TextFieldScreen.spec.js.snap

```diff
@@ -3933,7 +3933,7 @@ exports[`TextField Screen renders screen 1`] = `
                 accessible={false}
                 source={
                   {
-                    "testUri": "../../../src/assets/icons/xFlat.png",
```

Comment:
I think we should move the snap tests to the component's dir, they shouldn't be in the demo

Reply:
But then you'll be referencing the demo from the src code 
Also, focus on the assets change, if you decide to do other changes, do it in another PR please

Reply:
Yes, we do it already in other snaps and in private as well. I think we better keep tests in src instead of the demo